packages/components/src/ExpandingMenu/ExpandingMenu.stories.tsx

```diff
@@ -59,53 +46,6 @@ export default {
   }
 } as Meta
 
-const menuItems = [
-  {
-    icon: <DeclarationBlack />,
-    iconHover: <DeclarationBlue />,
-    label: 'Declarations',
-    onClick: () => alert('on Click Event')
-  },
```

Comment:
The hamburger opens up to an empty sidebar in the story

Reply:
@Zangetsu101 @jpye-finch  I had to remove unused code from here due to ESLint rules. We'll need to look at this in a different PR, and to see whats the relationship between ExpandingMenu and ToggleMenu

Reply:
Ah ok